.husky/pre-commit

```diff
@@ -1,4 +1,2 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
```

Comment:
Do we still need this? If so, what does it do?

Reply:
Good catch! I think we delete this and remove the `husky` dependency